packages/kbn-rule-data-utils/src/default_alerts_as_data.ts

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { ValuesType } from 'utility-types';
+
+const KIBANA_NAMESPACE = 'kibana' as const;
+const ALERT_NAMESPACE = `${KIBANA_NAMESPACE}.alert` as const;
+const ALERT_RULE_NAMESPACE = `${ALERT_NAMESPACE}.rule` as const;
+
+// kibana.space_ids - space ID(s) of the rule that created this alert
+const SPACE_IDS = `${KIBANA_NAMESPACE}.space_ids` as const;
+
+// kibana.version - Kibana version that this alert was created
+const VERSION = `${KIBANA_NAMESPACE}.version` as const;
+
+// kibana.alert.action_group - framework action group ID for this alert
+const ALERT_ACTION_GROUP = `${ALERT_NAMESPACE}.action_group` as const;
+
+// kibana.alert.duration.us - alert duration in nanoseconds - updated each execution
+// that the alert is active
+const ALERT_DURATION = `${ALERT_NAMESPACE}.duration.us` as const;
+
+// kibana.alert.end - timestamp when the alert is auto-recovered by the framework
+const ALERT_END = `${ALERT_NAMESPACE}.end` as const;
+
+// kibana.alert.flapping - whether the alert is currently in a flapping state
+const ALERT_FLAPPING = `${ALERT_NAMESPACE}.flapping` as const;
+
+// kibana.alert.id - alert ID, also known as alert instance ID
+const ALERT_ID = `${ALERT_NAMESPACE}.id` as const;
+
+// kibana.alert.reason - human readable reason that this alert is active
+const ALERT_REASON = `${ALERT_NAMESPACE}.reason` as const;
+
+// kibana.alert.start - timestamp when the alert is first active
+const ALERT_START = `${ALERT_NAMESPACE}.start` as const;
+
+// kibana.alert.status - active/recovered status of alert
+const ALERT_STATUS = `${ALERT_NAMESPACE}.status` as const;
+
+// kibana.alert.time_range - time range of alert from kibana.alert.start to now
+const ALERT_TIME_RANGE = `${ALERT_NAMESPACE}.time_range` as const;
+
+// kibana.alert.uuid - unique ID for the active span of this alert
+const ALERT_UUID = `${ALERT_NAMESPACE}.uuid` as const;
+
+// kibana.alert.workflow_status - open/closed status of alert
+const ALERT_WORKFLOW_STATUS = `${ALERT_NAMESPACE}.workflow_status` as const;
+
+// kibana.alert.rule.category - rule type name for rule that generated this alert
```

Comment:
Not sure what the right answer should be! Since we know that detection rules will copy over source ECS data for detection alerts, it's possible that if we are populating or aliasing the top level field, we might be overwriting source information? Regardless, since adding those fields or aliases is an additive change, I think we can add them later if we decide that is necessary.